src/Commands/RunRepeater.ts

```diff
@@ -114,7 +114,7 @@ export class RunRepeater implements CommandModule {
             });
         }
       })
-      .option('experimental-connection-reuse', {
+      .option('ntlm', {
```

Comment:
You should have left the existing name to avoid breaking changes; use the alias instead.